python/pyarrow/tests/test_extension_type.py

```diff
@@ -267,6 +267,20 @@ def test_ext_type__storage_type():
     assert ty.__class__ is ParamExtType
 
 
+def test_ext_type__byte_width():
+    ty = UuidType()
+    assert ty.storage_type.byte_width == 16
+    ty = ParamExtType(5)
+    assert ty.storage_type.byte_width == 5
+
+
+def test_ext_type__bit_width():
```

Comment:
```suggestion
def test_ext_type_bit_width():
```

Reply:
When I looked at the function nearby, I understood that the notation (`__`) was a function for testing `@property`.

After reading the review and searching the whole thing, I realized it was a typo and modified the other functions I used as a reference.

Thank you for review!